### PR TITLE
cockroach start and quit commands

### PR DESCRIPTION
**Pull request by jseldess**
Drafted a page listing all cockroach commands and well as pages explaining `cockroach start` and `cockroach quit`. Bunch of other smaller formatting and Jekyll-specific stuff in here, too, which you can disregard. 

@spencerkimball, @petermattis, @bdarnell, @mberhault, @BramGruneir: I'm most interested in feedback on the start and quit docs. Like this basic structure for command docs? Does the level of detail seem right? In the `start` docs, I still need to finish the advanced flags and examples, but everything else should be there. 

Probably easiest to look at the pages here:
[Cockroach Commands](http://cockroach-draft-docs.s3-website-us-east-1.amazonaws.com/cockroach-commands.html)
[Start a Node](http://cockroach-draft-docs.s3-website-us-east-1.amazonaws.com/start-a-node.html)
[Stop a Node](http://cockroach-draft-docs.s3-website-us-east-1.amazonaws.com/stop-a-node.html)

### CODE REVIEW

File: start-a-node.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<div id="toc"></div>
+
+## Synopsis
+
+~~~ shell
+# Run the command:
+$ ./cockroach start <flags>
+
+# View help directly in your shell:
+$ ./cockroach start --help
```
Comment (spencerkimball) [inline start-a-node.md]: Usually to view help on a command you'd instead write:

`$ ./cockroach help start`

File: stop-a-node.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+<div id="toc"></div>
+
+## Synopsis
+
+~~~ shell
+# Run the command:
+$ ./cockroach quit <flags>
+
+# View help directly in your shell:
+$ ./cockroach quit --help
```
Comment (spencerkimball) [inline stop-a-node.md]: `$ ./cockroach help quit`

File: stop-a-node.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+## Flags
+
+The `quit` command supports the following flags as well as [global flags](cockroach-commands.html#global-flags) that can be set on any command.
+
+
+Flag | Description 
+-----|------------
+`--certs` | The path to the directory containing the node's [security certificates](create-security-certificates.html). If the node was started with security (i.e., without the `--insecure` flag), the `--certs` flag is required. <br><br> **Default:** certs
+`--host` | A valid address for reaching the node. If the node advertises itself on multiple addresses (internal hostname, internal ip, external hostname, external ip, etc.), this can be any one of them. <br><br>**Default:** localhost
+`--insecure` | Whether or not the cluster is secure (authentication and encrypted client/node and inter-node communication). If the cluster is secure, set the `--certs` flag but leave this flag out. If the cluster is insecure, set this flag.
```
Comment (spencerkimball) [inline stop-a-node.md]: I believe we can omit the `--insecure` flag here (as soon as @mberhault finishes his change anyway – Marc, you should probably remove the inclusion of the `--insecure` flag in cli/flags.go for quit as part of your security change).

**Comment by spencerkimball** [comment]
LGTM

File: stop-a-node.md (hunk 3 above)
Comment (bdarnell) [inline stop-a-node.md]: The `quit` command will still require the `--insecure` flag when talking to an insecure node that is not running on localhost. (or do we even want to allow that with the CLI?)

File: cockroach-commands.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+`debug` | Extract data from files of a failed process.
+
+## Global Flags
+
+Flag | Value | Description
+-----|-------| -----------
+`--alsologtostderr` | | Log to standard error as well as files.
+`--color` | | Colorize standard error output according to severity (default "auto").
+`--log-backtrace-at` | |When logging hits line file:N, emit a stack trace (default :0).
+`--log-dir` | | If non-empty, write log files in this directory (default "/var/folders/5t/1rt6nzr17sjg87mz5n0fw15c0000gn/T/").
```
Comment (bdarnell) [inline cockroach-commands.md]: The garbage value here varies from one run to another; we should avoid including that in the docs.

**Comment by bdarnell** [comment]
The docs for `start` should discuss running in the background with `&`; if the node is started in the foreground then you would kill it with `C-c` instead of `cockroach quit`.

File: start-a-node.md (hunk 1 above)
Comment (jseldess) [inline start-a-node.md]: Thanks, I'll be sure to use this formulation in the docs.

File: stop-a-node.md (hunk 3 above)
Comment (jseldess) [inline stop-a-node.md]: I'll have to revise the docs in a bunch of places following's @mberhault's changes, so if it's ok with you all, I'll just open another pull request for that.

File: cockroach-commands.md (hunk 4 above)
Comment (BramGruneir) [inline cockroach-commands.md]: And the default log directory is a sub directory of the default store directory.  So it should be /cockroach-data/logs or something like that.  The default value in the CLI should be updated to either show this or nothing at all.  Having a random temp directory is not good.

Comment (jseldess) [inline cockroach-commands.md]: Thanks, all. Just chatted with @petermattis about this a bit. He explained the above, but I need an introduction to how standard output, standard error, and logs all work before it makes sense to rework the language here. I'll just remove for now and open another pull request to deal with these topics. 

And I opened an issue with new help text for `--log-dir`: https://github.com/cockroachdb/cockroach/issues/4816

**Pull request merged**